src/api/instance-methods.md

```diff
@@ -14,7 +14,7 @@
 
 - **Usage:**
 
-  Watch an expression or a computed function on the Vue instance for changes. The callback gets called with the new value and the old value. The expression only accepts dot-delimited paths. For more complex expressions, use a function instead.
+  Watch a string expression or a computed function on the Vue instance for changes. The callback gets called with the new value and the old value. The string expression only accepts top-level data and component property names. For more complex expressions, use a function instead.
```

Comment:
I believe by "expression", they mean via function:`this.$watch(() => this.c.d, ...)`

And by "computed function on the Vue instance", they mean watch a computed property on the Vue instance via property name (string).

Reply:
It sounds like that, but it is conflicting with `The expression only accepts dot-delimited paths`. I belive the suggestions below makes everything clearer.

Reply:
It should be clearer with the latest changes, thanks for the explanation. 👍